src/requestVariableCacheValueProcessor.ts

```diff
@@ -0,0 +1,109 @@
+"use strict";
+import { window } from "vscode";
+
+import { MimeUtility } from "./mimeUtility";
+import { RequestVariableCacheValue } from './models/requestVariableCacheValue';
+import { HttpResponse } from './models/httpResponse';
+
+const validPathRegex = new RegExp(/(\w+)(\.\w+|\[\d+\])*/);
+const partRegex = new RegExp(/(\w+)(\[\d+\])*/);
+const arrayIndexRegex = new RegExp(/\[(\d+)\]/);
+
+export class RequestVariableCacheValueProcessor {
+    public static getValueAtPath(value: RequestVariableCacheValue, path: string): any {
+        if (!value
+            || !path
+            || !RequestVariableCacheValueProcessor.isValidPath(path)) {
+            return undefined;
+        }
+
+        const parts = path.split(".");
```

Comment:
Can we simply use array destruction syntax here, which seems more readable than `parts` array, i.e.
```js
const [name, source, type, ...rest] = path.split(".");
```

Reply:
Sure!